engine/table/src/main/java/io/deephaven/engine/table/impl/QueryTable.java

```diff
@@ -1314,22 +1316,6 @@ void handleUncaughtException(Exception throwable) {
                 });
     }
 
-    @SuppressWarnings("WeakerAccess")
-    protected WritableRowSet filterRows(RowSet currentMapping, RowSet fullSet, boolean usePrev,
```

Comment:
This was unused?

Reply:
Yes, QueryTable#filterRows exists as ConjunctiveFilter#andImpl now. I came across the snippet when looking for for-loops that manage WhereFilters.